README.md

```diff
@@ -47,9 +91,9 @@ Let's say we have a table "post" with a field "author_id". author_id contains a
 ```javascript
 Arrow.Model.extend('article', {
 	fields: {
-		title: { type: String, model: 'post' },
-		content: { type: String, model: 'post' },
-		author_id: { type: Number, model: 'post' },
+		title: { type: String, model: 'post', name: 'title' },
```

Comment:
"name" is for consistency. If we say that we need name when selecting specific fields from a model, then while it's not required for types other than array and object, it makes sense to add it as best practice. That means whenever it's not there it's clearer that the field is containing something other than a specific field aka "title" from "post".